crt_portal/static/sass/custom/table.scss

```diff
@@ -20,11 +20,9 @@
   }
 }
 
-.tr-status-new {
-  td {
-    font-weight: bold;
-    text-transform: uppercase;
-  }
+.status-new {
```

Comment:
I think the uppercase transform needs to move to `status-tag`  so open and closed are also capitalized